.gitattributes

```diff
@@ -1,11 +1,13 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
 * text=auto
 
+MacOS_users_read_this_first.txt eol=lf
```

Comment:
Moving the exceptions towards the top, so they stand out more (I was hoping order would matter and rules would take precedence, not the case)